### DISCUSSION TITLE
Testplane - логирование упавших тестов.

### DISCUSSION THREAD

**Discussion by zi-klon** [gemini-testing/testplane#983] [Q&A] [answered]
Привет! Была произведена миграция с **Hermione** 8 версии на **Testplane**. Теперь после выполнения тестов, если были тесты с ошибками, то в корень проекта в папку `.testplane` пишется файл `failed.json` с информацией по упавшим тестам. Раньше такого не было. Можно ли каким-то образом переопределить путь, куда будет писаться этот файл или совсем отключить эту опцию? В новой документации по **Testplane**, к сожалению, ничего не смог найти.

**Comment by sipayRT** [comment, marked as answer]
Привет

Да, в последних версиях мы действительно добавили возможность быстрого перезапуска упавших тестов и сохраняем в этой папке json-отчет. Опции про отключение сохранения этого отчета нет, но есть возможность переопределить путь к отчету. Все доступные опции доступны в документации на gh - https://github.com/gemini-testing/testplane/blob/master/docs/config.md#lastfailed. Сегодня донесем эту информацию и на https://testplane.io/

**Reply by zi-klon** [reply]
Спасибо за оперативный ответ!
